code/modules/mob/living/silicon/ai/ai_say.dm

```diff
@@ -129,7 +129,7 @@
 
 	log_message("made a vocal announcement with the following message: [message].", LOG_GAME)
 	log_talk(message, LOG_SAY, tag="VOX Announcement")
-	say(";[message]", forced = "VOX Announcement")
+	minor_announce(capitalize(message), "[name] announces:", sound_override = TRUE)
 
 	for(var/word in words)
 		play_vox_word(word, src.z, null)
```

Comment:
Seems like the VOX message only plays for the AI's z-level which I never realized. Should I change the announcement so it only gets sent to people on the AI's z-level?